app/src/main/res/layout/activity_view_annotation_animation.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.mapbox.maps.MapView
```

Comment:
As we have just one element in the layout and it's `MapView` - perhaps it's easier to create in code making the activity code fully isolated and re-usable. Then this xml file could be totally removed.